tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageViewModel.cs

```diff
@@ -75,13 +76,15 @@ public partial class PackageViewModel : ObservableObject
         IWinGetPackage package,
         IThemeSelectorService themeSelector,
         IScreenReaderService screenReaderService,
-        IHost host)
+        IHost host,

```

Comment:
You can remove host here and just keep the orchestrator

Reply:
sure, I can update this in a future PR